src/dist/replication/lib/replica_stub.h

```diff
@@ -186,18 +186,14 @@ class replica_stub : public serverlet<replica_stub>, public ref_counter
     typedef std::function<void(::dsn::replication::replica *rep)> local_execution;
 
     // This function is used for partition split, caller(replica)
-    // - case1. parent want child execute <handler>, child will execute <handler> if child is
-    // valid(<pid>.app_id>0) and existed, otherwise parent will execute <error_handler>
-    // - case2. child want parent execute <handler>, parent will execute <handler> if parent
-    // exist, otherwise child will execute <error_handler>
-    void split_replica_exec(gpid pid,
-                            local_execution handler,
-                            local_execution error_handler,
-                            gpid error_handler_gpid);
-
-    // This function is used for partition split error handler, caller(replica)
-    // if partition split meet error, parent/child may want child/parent execute error handler
-    // if replica <pid> valid and exist, execute <handler>, otherwise return
+    // parent/child may want child/parent to execute function during partition split
+    // for parent, if replica existed, will execute function {handler} and return ERR_OK, else
+    // return ERR_OBJECT_NOT_FOUND
+    // for child, if pid.app_id > 0 and existed, will execute function {handler} and return ERR_OK,
+    // else return ERR_OBJECT_NOT_FOUND
+    dsn::error_code split_replica_exec(dsn::task_code code, gpid pid, local_execution handler);
```

Comment:
This comment makes me more confused. I still don't know why it makes a difference whether the replica is child or parent.

```
// If replica `pid` exists, will execute function `handler` and return ERR_OK, 
// return ERR_OBJECT_NOT_FOUND otherwise.
```